src/vm/man/vmadm.1m.md

```diff
@@ -337,6 +337,12 @@ tab-complete UUIDs rather than having to type them out for every command.
         If vmadmd sees a VM that has a transition but reaches the expiry before
         actually turning off, it re-sends the stop command with the -F option.
 
+        For Docker VMs, vmadm will send a SIGTERM to init and then wait some
+        number of seconds for the init process to exit. If it has not exited by
+        the timeout expiry, a SIGKILL will be sent. The default timeout is 10
+        seconds but this can be adjusted with the -t <timeout seconds> option.
+        For non-Docker VMs the -t option will do nothing.
```

Comment:
I realize that history of vmadm is silently ignoring unknown arguments. But it seems like if I'm a user of other types of VMs the fact that it's silently ignored seems odd. Is there a reason we shouldn't make it an error to specify it if it's not actually supported for a given VM type?